napari/_qt/widgets/_tests/test_qt_tooltip.py

```diff
@@ -18,6 +21,9 @@ def test_qt_tooltip_label(qtbot):
     widget.show()
 
     assert QToolTip.text() == ""
-    qtbot.mouseMove(widget)
+    # put mouse outside of widget
```

Comment:
It looks like the code is simulating an enter event where the mouse cursor is in the center of the widget, so this comment seems inaccurate.